examples/benchmark.py

```diff
@@ -182,46 +192,56 @@ def run(self, *args, **kwargs):
             warning("Could not load any results, nothing to plot. Exiting...")
             sys.exit(0)
 
-        gflopss = bench.lookup(params=parameters, measure="gflopss", event="loop_body")
-        oi = bench.lookup(params=parameters, measure="oi", event="loop_body")
-        time = bench.lookup(params=parameters, measure="timings", event="loop_body")
+        gflopss = bench.lookup(params=parameters, measure="gflopss", event="main")
+        oi = bench.lookup(params=parameters, measure="oi", event="main")
+        time = bench.lookup(params=parameters, measure="timings", event="main")
```

Comment:
Is this backwards compatible? I'm assuming we want to run 2.0.1 vs. 3.0 in like-for-like performance comparisons, so should we have a switch here?